src/test/ui/asm-out-assign-imm.stderr

```diff
@@ -9,4 +9,5 @@ LL |         asm!("mov $1, $0" : "=r"(x) : "r"(5));
 
 error: aborting due to previous error
 
-If you want more information on this error, try using "rustc --explain E0384"
+error: For more information about this error, try `rustc --explain E0384`.
```

Comment:
Is it possible to get rid of this `error: ` in the beginning? This is not an error.